ecs-cli/modules/commands/cluster/cluster_command.go

```diff
@@ -84,7 +84,7 @@ func clusterUpFlags() []cli.Flag {
 		},
 		cli.StringFlag{
 			Name:  flags.InstanceTypeFlag,
-			Usage: "[Optional] Specifies the EC2 instance type for your container instances. Defaults to t2.micro. NOTE: Not applicable for launch type FARGATE.",
+			Usage: "[Optional] Specifies the EC2 instance type for your container instances. If you specify the A1 instance family, the ecs optimized Arm AMI will be used, otherwise the x86 AMI will be used. Defaults to t2.micro. NOTE: Not applicable for launch type FARGATE.",
```

Comment:
Capitalize ECS. And maybe Arm64? Also, will this be Linux 2 or is that a separate parameter?

Reply:
We use Amazon Linux 2 for both architectures

Reply:
@petderek @jahkeup What's the best way to refer to architecture- Arm, arm64, Arm64, ARM64?

Reply:
We refer to it as `arm64` in some places on the EC2 console and naming of the AMIs. The ECS agent also refers to this architecture as `arm64`.

There are several ways to refer to the hardware platform, but consistency here is good for users.

Reply:
OCI and golang use 'arm64' when referring to the architecture.

"Arm"  has been the company's brand in recent years -- although it was originally an upper-case acronym.